src/Sulu/Component/DocumentManager/Tests/Unit/NodeHelperTest.php

```diff
@@ -79,35 +80,37 @@ public function testMoveWithDestinationName(): void
 
     public function testCopy(): void
     {
-        /** @var Workspace $workspace */
+        /** @var ObjectProphecy<Workspace> $workspace */
         $workspace = $this->prophesize(Workspace::class);
-        $this->session->getWorkspace()->willReturn($workspace->reveal());
+    $this->session->getWorkspace()->willReturn($workspace->reveal());
+        $identifier = UUIDHelper::generateUUID();
 
         $destinationNode = $this->prophesize(NodeInterface::class);
         $destinationNode->getPath()->willReturn('/path/to/some/other/node');
-        $this->session->getNodeByIdentifier('uuid')->willReturn($destinationNode->reveal());
+        $this->session->getNodeByIdentifier($identifier)->willReturn($destinationNode->reveal());
```

Comment:
As discussed in the main PR: This only worked because the prophecy extension never checked the method calls. fixed them now.